app/viewmodels/govuk/FieldsetFluency.scala

```diff
@@ -60,11 +60,17 @@ trait FieldsetFluency {
 
   implicit class FluentLegend(legend: Legend) {
 
+    //todo: shell we keep both?
     def asPageHeading(size: LegendSize = LegendSize.Large): Legend =
       legend
         .copy(isPageHeading = true)
         .withCssClass(size.toString)
 
+    def asPageHeading(isPageHeading: Boolean, size: LegendSize): Legend =
```

Comment:
why do we need this one? if we dont want this to be a page heading dont call the method?

Reply:
yes, that would be just create a legend with the right class and content, but wanted to add another way to use the LegendViewModel, but I see the point.